src/advisors/dep_check.ts

```diff
@@ -145,26 +150,63 @@ class DepCheckAdvisor implements IAdvisor {
                     }
                   )
                 );
+                // only resolved packages that are ported by
+                // emscripten and the compiler and linker flags for
+                // this package are added can be added into
+                // resolvedPkgs
+                resolvedPkgs.push(pkgName);
               } else {
                 nonPorts.push(m.groups.pkg);
               }
+            } else {
+              const n = lines[i].match(libraryReg);
+              const o = lines[i].match(includeDirReg);
```

Comment:
We may want to avoid naming variables to `n` and `o` as much as possible.

Reply:
Got it, working on it